README.md

```diff
@@ -4,10 +4,13 @@ Our vision is an open source software ecosystem where the time to fix a vulnerab
 
 ## Objectives and Key Results (CY 2020)
 
-The first objectives we're using to track our progress towards that vision are:
+The OpenSSF Vulnerability Disclosures Working Group seeks to help improve the overall security of the open source software ecosystem by helping mature and advocate well-managed vulnerability reporting and communication.  We plan on address this challenge through the following actions:
```

Comment:
super nitpick, but consider removing "help" before "improve", not necessary for your meaning: 
`The ... Working Group seeks to improve the overall security of the open source software ecosystem by helping mature...`

Reply:
maybe "develop" in place of "mature"?  `...by helping develop and advocate well-managed...`

Reply:
verb tense, `We plan on addressing this challenge...`